filter/timestamp_range_test.go

```diff
@@ -50,6 +51,20 @@ func TestTimestampRange(t *testing.T) {
 			endDate:   "2018-02-06 09:34:01",
 			want:      false,
 		},
+		{
+			name:      "start now",
+			ts:        int(time.Now().Unix()),
+			startDate: "now",
+			endDate:   "2122-02-06 09:34:01",
+			want:      true,
+		},
+		{
+			name:      "end now",
+			ts:        int(time.Now().Unix()),
+			startDate: "2018-02-06 09:34:01",
+			endDate:   "now",
+			want:      false,
+		},
```

Comment:
would be nice to see a test with `"now"` for both bounds

Reply:
done